README.md

```diff
@@ -4,9 +4,248 @@ This guide will show how Rust programs can be cross compiled for the Raspberry
 Pi using Cargo. These instructions may or may not work for your particular
 system, so you may have to adjust the procedure to fit your configuration.
 
-You can build the cross compiler and your rust project...
-* ...by hand on your host machine -- flexible but also more complex ([see MANUAL.md](MANUAL.md))
-* ...utilizing a docker image and running inside a docker container -- simple but a little less flexible ([see DOCKER.md](DOCKER.md))
+There exists also a docker container version of the Rust cross-compiler
+for the Raspberry: https://github.com/Ragnaroek/rust-on-raspberry-docker.
```

Comment:
Would you mind just changing "There exists also..." to "There is also..."? It's otherwise great!